test-fixtures/src/bin/upstream-async.rs

```diff
@@ -2,7 +2,7 @@
 //! on `PendingRequest`s.
 //!
 //! The test makes async requests to two different backends, each of which returns an identifying header,
-//! and then checks that both respones are eventually returned and processed.
+//! and then checks that both response are eventually returned and processed.
```

Comment:
I think this one is meant to be plural as it is checking multiple responses

Reply:
Thanks, fixed in https://github.com/fastly/Viceroy/pull/166/commits/ba945cc77caa2e65585d25ef5f81dc375a4070b3